.github/workflows/image-build-binary.yml

```
@@ -0,0 +1,93 @@
+## Github workflow to build a multiarch docker image from pre-built binaries
+
+name: Docker Image (Binary)
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+        description: "Version tag for alpine images"
+      docker-org:
+        required: false
+        type: string
+        description: "Docker repo org for uploading images (defaults to github org)"
+        default: "${GITHUB_REPOSITORY_OWNER}"
+
+defaults:
+  run:
+    shell: bash
+
+## Define which docker arch to build for
+env:
+  docker_platforms: "linux/arm64, linux/arm/v7, linux/amd64, linux/amd64/v2, linux/amd64/v3"
+  docker-org: blockstack
+
+concurrency:
+  group: docker-image-binary-${{ github.head_ref || github.ref || github.run_id }}
+  ## Always cancel duplicate jobs
+  cancel-in-progress: true
+
+run-name: ${{ inputs.tag }}
+
+jobs:
+  ## Runs when the following is true:
+  ##  - tag is provided
+  ##  - workflow is building default branch (master)
+  image:
+    if: |
+      inputs.tag != '' && 
+      github.ref == format('refs/heads/{0}', github.event.repository.default_branch)
+    name: Build Image
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      ## Build a maximum of 2 images concurrently based on matrix.dist
+      max-parallel: 2
+      matrix:
+        dist:
+          - alpine
+          - debian
+    steps:
+      ## Setup Docker for the builds
+      - name: Docker setup
+        uses: stacks-network/actions/docker@main
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      ## Set docker metatdata
+      ## - depending on the matrix.dist, different tags will be enabled
+      ## ex. alpine will have this tag: `type=ref,event=tag,enable=${{ matrix.dist == 'alpine' }}`
+      - name: Docker Metadata ( ${{matrix.dist}} )
+        id: docker_metadata
+        uses: docker/metadata-action@96383f45573cb7f253c731d3b3ab81c87ef81934 #v5.0.0
+        with:
+          images: |
+            ${{env.docker-org}}/${{ github.event.repository.name }}
+            ${{env.docker-org}}/stacks-core
```

Correct me if I'm wrong, but I believe this will result in a list of:
```
stacks-network/stacks-core
stacks-network/stacks-core
```

Instead, did you intend this to continue pushing to the old Docker repo?

```suggestion
            ${{env.docker-org}}/${{ github.event.repository.name }}
            ${{env.docker-org}}/stacks-blockchain
```

---

i believe you are correct - the repo name change threw a wrench in things. thanks!